dmotion:
  ankle_z:
    data: [0, 0.90000000000000002, 1.1694222794363318, 1.3854888412358031, 1.7873105212155864, 2.0688970459709859, 2.3570949134766668, 3.7307293685265557, 4.1279220717359202, 5.5, 0, 1.8000000000000009, 5.0001199267573604, 8.2186760120718247, 12.839020953215089, 9.3801586190519277, 12.145800303455054, 8.403217157837318, 7.9772975900832694, 0, 0, 8.1443464279742788, 9.2051564333252518, 15.055722724477155, 0.4265361461391558, -1.2799416321930757, 1.92098212697116, -0.29052685673191875, -2.7776068539149659, -0.1583844403245363]